pkgs/development/tools/misc/pkg-config/default.nix

```diff
@@ -1,9 +1,10 @@
-{stdenv, fetchurl, libiconv, vanilla ? false }:
+{ stdenv, fetchurl, libiconv, vanilla ? false }:
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
-  name = "pkg-config-0.29.2";
+  name = "pkg-config-${version}";
+  version = "0.29.2";
```

Comment:
Leaving the `version` attribute in the derivation saves one line for users who are going to pin it in an overlay, since they don't need to update the `name` attribute as well.

Reply:
Really?  I wouldn't think that works without some additional "knot-tying" (i.e. using version from the `self` attribute).

Reply:
Ah yes you're right, I was confused.  Still I think it's nice to have the version attached to the derivation as a separate attribute, as it makes it easier to inspect the package's version mechanically.

Reply:
Can't one use pname for the name-without-version now so we do get the delayed binding?

Reply:
yes, we can

Reply:
Aha, I've mostly been working with python overlays, which have pname, which follows this nice behavior; glad to see it's being more widely adopted in NixPkgs in general :+1: 

I updated this PR to use it, slimmed down the urls to just the canonical release one, and enabled parallel building since it appears to not have concurrency issues.